### PR TITLE
Provide 4.9 jobs for the PAO

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: pao-src-oc
+operator:
+  bundles:
+  - dockerfile_path: openshift-ci/Dockerfile.bundle.ci
+  - dockerfile_path: openshift-ci/Dockerfile.bundle-4.7.ci
```
Comment (yanirq) [inline ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml:63]: is openshift-ci/Dockerfile.bundle-4.7.ci  the right docker bundle file ?  (not 4.8?)

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: optional-operators-ci-gcp
+- as: e2e-gcp-operator-upgrade
+  steps:
+    cluster_profile: gcp
+    env:
+      OO_CHANNEL: "4.7"
```
Comment (yanirq) [inline ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml:107]: change channel to 4.8?

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml (hunk 1 above)
Comment (cynepco3hahue) [reply ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml:63]: yep, it for the upgrade test
so 
`openshift-ci/Dockerfile.bundle.ci` - build 4.8 bundle
`openshift-ci/Dockerfile.bundle-4.7.ci` - build 4.7 bundle

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml (hunk 2 above)
Comment (cynepco3hahue) [reply ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.8.yaml:107]: it for the upgrade test, we start with the 4.7 and upgrade to 4.8 under the tets

**Pull request merged**